workspaces/cli-config/src/helpers/default-ignore-rules.ts

```diff
@@ -17,6 +17,6 @@ export const defaultIgnoreRules = [
 
 export const defaultIgnoreFile = `
 # Default Ignore Rules
-# Learn to configure your own at http://localhost:4000/docs/using/advanced-configuration#ignoring-api-paths
+# Learn to configure your own at http://localhost:4000/reference/optic-yaml/ignore
```

Comment:
Do we want this to point to the hosted `useoptic.com` link? Since that will probably be up to date - or do we want this to match the current locally running cli?

Reply:
ooh, nice catch, thanks! Fixing now.